includes/Generator/Plugins/Twitter.php

```diff
@@ -66,8 +58,8 @@ public function addMetadata(): void {
 		$this->outputPage->addHeadItem(
 			'twitter:card', Html::element(
 				'meta', [
-				self::$htmlElementPropertyKey => 'twitter:card',
-				self::$htmlElementContentKey => $twitterCardType,
+					self::$htmlElementPropertyKey => 'twitter:card',
+					self::$htmlElementContentKey => $twitterCardType,
 				]
 			)
 		);
```

Comment:
We can savely change calls to `addHeadItem` with `addMeta('twitter:card', $twitterCardType)`.

`addMeta` automatically creates meta tags with a `name` key, addHeadItem is needed when we want to create meta tags with a `property` key.